docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Internationalization (i18n) - Strapi Developers Documentation
+description: Instructions on how to use Strapi Content API with the Internationalization (i18n) optional plugin
+---
+
+# Internationalization (i18n)
+<!-- ? wondering if it would be a good idea to add the 🌍 icon to the h1 — if yes, we should do this for the h1 of other plugins as well -->
+
+> Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language.
+>
+> -- <cite>[W3C](https://www.w3.org/International/questions/qa-i18n.en#i18n)</cite>
+
+<!-- ? not sure if using a citation is a good idea to introduce/define i18n — feedback is welcome! -->
+
+The Internationalization (i18n) plugin allows Strapi users to create, manage and distribute content in several languages.
+It's an optional plugin that you can [install from the Marketplace](/user-docs/latest/plugins/installing-plugins-via-marketplace.md).
+<!-- TODO: add programmatic instructions to install plugin? like strapi install i18n-->
+
+The i18n plugin:
+
+- allows admin panel users to create several localized versions of their content (see [user guide](/user-docs/latest/content-manager/translating-content.md))
+- allows developers to build localized projects by fetching and consuming the right content depending on the country/language of the audience.
+
+::: tip NOTE
+The i18n plugin does not translate the users' content nor adapt the admin interface to languages specificities (like displaying the admin panel in Right To Left (RTL) format).
+:::
+
+The i18n plugin adds new features to the [Content API](/developer-docs/latest/developer-resources/content-api/content-api.md):
+
+- a new [`locale`](#getting-localized-entries-with-the-locale-parameter) parameter to fetch content only for a specified locale
+- the ability to create a localized entry, either [from scratch](#creating-a-new-localized-entry) or [for an existing localized entry](#creating-a-localization-for-an-existing-entry)
+
+## Getting localized entries with the `locale` parameter
```

Comment:
Maybe we should update the content-api doc or include a table for it here also :thinking: 

http://localhost:8080/documentation/developer-docs/latest/developer-resources/content-api/content-api.html#api-endpoints

Reply:
Thanks Derrick. This is something I already discussed with @Convly 
My first draft actually modified the Content API docs ;-)
But we would like to avoid disseminating "plugins-only" information into the core Content API docs. The idea is to mimic the product behavior: if we remove a plugin, it should not impact the "core" (docs).

However, I could probably simply mention in the core Content API doc that i18n adds some features and link to the /plugins/i18n.md file from there. Will suggest something like this soon ;-)

Reply:
Suggestion:
See [this commit](https://github.com/strapi/documentation/pull/224/commits/274c2ba6899f1859c3bc578bf8f9de721943dca0)

Reply:
I'm not really a big fan of citing an optional plugin directly from the generic/core documentation, I just feel it adds some complexity & overhead. But maybe stating the params "if i18n is activated" is a good compromise :man_shrugging: